lib/src/sai_redis_generic_get_stats.cpp

```diff
@@ -0,0 +1,201 @@
+#include "sai_redis.h"
+#include "meta/saiserialize.h"
+
+sai_status_t internal_redis_get_stats_process(
```

Comment:
This is not an acceptable coding style.

You can write your function like so:

sai_status_t internal_redis_get_stats_process(_In_ sai_object_type_t object_type,
                                                                            _In_ uint32_t count,
                                                                            _Out_ uint64_t *counter_list,
                                                                            _In_ swss::KeyOpFieldsValuesTuple &kco)

or

sai_status_t internal_redis_get_stats_process(_In_ sai_object_type_t object_type, _In_ uint32_t count,
                                                                            _Out_ uint64_t *counter_list,  _In_ swss::KeyOpFieldsValuesTuple &kco)

but not what you have.

Same hold for other similar functions in your diffs.

Reply:
Can you please explain what's the difference between functions in this file and https://github.com/Azure/sonic-sairedis/blob/master/lib/src/sai_redis_generic_create.cpp ?

Reply:
Of course - you're just pointing out that different engineers have different styles, which is expected. So since that style wasn't corrected at commit time, eventually people will follow it and we should go ahead and adopt it, hence ending up with different styles inside sonic. Makes perfect sense to me :-)

Reply:
Actually, I'm just trying to make it similar to what is already there, like

https://github.com/Azure/sonic-sairedis/blob/master/lib/src/sai_redis_buffer.cpp
https://github.com/Azure/sonic-sairedis/blob/master/lib/src/sai_redis_notifications.cpp
https://github.com/Azure/sonic-sairedis/blob/master/lib/src/sai_redis_policer.cpp
...

sairedis library has a common style across almost all its source files. There is, of course, some code that doesn't follow common style, but most of it follows the common pattern and I, in turn, follow it.

If you want some formal version of code style or you don't agree with how my function declarations look (which are same as almost all function declarations in sairedis), then I think that it's up to a discussion with the maintainer.